Trivago/Jade/Application/JsonApi/Request/RequestFactory.php

```diff
@@ -372,10 +385,10 @@ private function createRelationshipBag($data, $dataAsObject, $acceptNull)
         $relationships = [];
         foreach ($rawRelationships as $relationshipName => $rawRelationship) {
             if (!$resourceConfig->hasRelationship($relationshipName)) {
-                throw InvalidRequest::createWithMessage('data.relationships.'.$relationshipName, 'invalid_path', sprintf('No relationship on %s called %s', $data['type'], $relationshipName));
+                throw InvalidRequest::createWithMessage('data.relationships.' . $relationshipName, 'invalid_path', sprintf('No relationship on %s called %s', $data['type'], $relationshipName));
```

Comment:
Why did you suddenly changed all the concatenations in this file? 😮 
Let's keep this PR simple and just focus on merging what you initially started it for

Reply:
Hoo! Sorry. I have no idea of this changes, maybe when I used the function of automatically rearrange code of the PHPStorm, it changed all concatenations.

Is there a major issue with this? Do I have to rollback all rearranged code?

I done several changes into the code of this class, and doing a rollback to all that code would be painful.